rotkehlchen/chain/aggregator.py

```diff
@@ -1713,6 +1707,11 @@ def flush_eth2_cache(self) -> None:
         self.flush_cache('refresh_eth2_get_daily_stats')
         self.flush_cache('get_eth2_daily_stats')
         self.flush_cache('query_eth2_balances')
+        self.flush_cache('query_balances')
```

Comment:
Can you also change the docstring and write why you are all flushing also all balances cache?

Since this is anything that is touched by eth2 changes and needs to be invalidated.

You did not move this:`self.flush_cache('query_balances', blockchain=SupportedBlockchain.ETHEREUM_BEACONCHAIN)`. Why?